### ISSUE TITLE
options for testing

### ISSUE THREAD

**Issue by chandnikathuria1992** [kenshohara/3D-ResNets-PyTorch#52]
on running code for resnext101(fine-tuning resnext 101 kinetics pretrained model ) i got  clip accuracy around 86 %.
val.json file is created during testing as per ur code, now i want to get video accuracy, so first i need to to test by setting no_val and no_train =true and Test=true in opts.py.
i ran code with above options, it created val.json
on evaluating val.json its giving accuracy =0.28
do in neeed to change any other option in opts.py?

can please mention options for testing only using fine tuned model.

**Comment by chandnikathuria1992** [comment]
Thanks alot....
Using resume path resolved the issue..
